### PR TITLE
Fix outdated link in MetalInfo

### PR DESCRIPTION
**Pull request by lenguyenthanh**
The gitter link seems return 404. so I removed it and also added discord link in MetalInfo table.

### CODE REVIEW

File: lua/metals.lua [hunk 1]
```diff
@@ -156,7 +156,7 @@ M.info = function()
     end
     table.insert(output, "")
     table.insert(output, "## Helpful links")
-    table.insert(output, "  - https://gitter.im/scalameta/metals-vim")
+    table.insert(output, "  - https://discord.gg/FaVDrJegEh")
```
Comment (ckipp01) [inline lua/metals.lua:159]: Do you know if this link is one that expires after a certain amount of time, or is this one that was created under "expires never"? We want to make sure we're not including one that will expire.

Also, can we add another entry with the matrix bridge? So just add another line pointing to the matrix link.

https://matrix.to/#/#scalameta/vim-users:matrix.org

**Review by ckipp01 — CHANGES_REQUESTED** [review]
Thanks for sending this in! Just a quick note on the matrix link.

File: lua/metals.lua (hunk 1 above)
Comment (lenguyenthanh) [reply lua/metals.lua:159]: Added matrix: https://matrix.to/#/#scalameta:vim-users

**Review by ckipp01 — APPROVED** [review]
Thanks a lot for the update @lenguyenthanh!

LGTM!

**Pull request merged**